msvc/scripts/genproj.cs

```diff
@@ -944,15 +1029,13 @@ string GenerateStep (string library, string suffix, string eventKey)
 		if (q != -1)
 			target = target + Load (library.Substring (0, q) + suffix);
 
-		if (target.IndexOf ("@MONO@") != -1){
-			target_unix = target.Replace ("@MONO@", "mono").Replace ("@CAT@", "cat");
-			target_windows = target.Replace ("@MONO@", "").Replace ("@CAT@", "type");
-		} else {
-			target_unix = target.Replace ("jay.exe", "jay");
-			target_windows = target;
-		}
+		target_unix = target.Replace ("@MONO@", "mono").Replace ("@CAT@", "cat");
```

Comment:
Feels kinda lame that we are still dealing with this sort of thing. There is no in-proc mechanism w/o running command lines? Ok.